src/content/blog/flutter-3.32-release.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Flutter 3.32 Support and Review
+author: shorebirdtech
+description:
+  Updating Shorebird for the latest Flutter release and highlighting our
+  favorite updates.
+date: 2025-05-28
+cover: flutter-3.32-cover.png
+---
+
+We’re excited to announce that Shorebird now fully supports Flutter 3.32.0 and
+Dart 3.8! If you’re looking to update your Flutter apps with Shorebird, just
+upgrade to Shorebird v1.6.39 and you’ll be good to go. All it takes is a quick
+`shorebird upgrade` to ensure you’re running the latest version.
+
+We also wanted to take a moment to share our thoughts on these new releases.
+They bring significant enhancements that streamline cross-platform development
+and improve performance—not just for our customers, but for the entire Flutter
+community. While we won’t rehash everything (you can read the official
+[Flutter](https://medium.com/flutter/whats-new-in-flutter-3-32-40c1086bab6e) and
+[Dart](https://medium.com/dartlang/announcing-dart-3-8-724eaaec9f47) posts for
+the full details), we’d like to highlight a few of our favorite changes.
+
+## Look and Feel Improvements
+
+Google has introduced support for Apple “squircles” and made numerous
+enhancements to the Cupertino library, improving the iOS look and feel for
+Flutter apps. While some of these changes are subtle, they’re crucial for apps
+to look and feel their best. If you’re targeting iOS, your apps should feel even
+more polished for your users.
+
+Similarly, the Flutter team has made a wide range of improvements to Material
```

Comment:
Similarly, if handy.

Reply:
There wasn't anything super specific here from the Flutter team so going to skip an image. I think mentioning it is fine but there wasn't a standout thing like they did for squircle.